test/unit/auth/tenant-manager.spec.ts

```diff
@@ -388,6 +389,7 @@ describe('TenantManager', () => {
         enabled: true,
         passwordRequired: true,
       },
+      anonymousSignInEnabled: true,
```

Comment:
Do we have tests for where this is undefined or explicitly false?

Reply:
Yes, though they're in the integration tests. grep test/integration/auth.spec.ts for anonymousSignInEnabled. (Broadly, they start at line 809, though there's an update test on line 1265 that explicitly goes through all possibilities; undefined, true, false.)